smoke-tests/fake-backend/build.gradle

```diff
@@ -30,10 +30,8 @@ dependencies {
 }
 
 jib {
-  to {
-    image = "open-telemetry-docker-dev.bintray.io/java/smoke-fake-backend"
-  }
-  from {
-    image = "gcr.io/distroless/java-debian10:11"
-  }
+  container.creationTime = 'USE_CURRENT_TIMESTAMP'
```

Comment:
Is there a specific need for this change?

Reply:
It is super inconvenient, that by default images built by jib will have 0 as a creation timestamp, which breaks ordering in `docker images`, because all such images are created `50 years ago`. Issue is tagged `contributor experience`, so I find this change is needed.

Reply:
So you don't think Jib's desire for reproducible builds as convincing? :) Why ordering of `docker images` is important/inconvenient?

Reply:
Achieving the desire for reproducible builds by lying about image creation time is not really convincing, yes. On multiple occasions debugging/troubleshooting either in development or in production I have benefitted from correct creation time that `docker images` show, because then I can easily spot, that some build step somewhere has failed silently and I'm using old images, that explain why something is not working as expected.

Reply:
But now you have build time in image tag. Isn't it enough?

What happens when you run `./gradlew jibDockerBuild` two times in a row with `USE_CURRENT_TIMESTAMP` and without? Will it create two docker images in both cases or not?

Reply:
If you don't change anything, and will just rebuild, it will not build a new image in either case. You will have two tags pointing at the same image ID. It only matters, if you do a change in some source file like A -> B -> A. Then in case of creation timestamp zero you will get 2 image ID-s, one for A and one for B. But in case of `USE_CURRENT_TIMESTAMP` you will get 3 distinct image IDs.